include/tbl/vsl_tags.h

```diff
@@ -138,15 +138,17 @@ SLTM(SessClose, 0, "Client connection closed",
 SLTM(BackendOpen, 0, "Backend connection opened",
 	"Logged when a new backend connection is opened.\n\n"
 	"The format is::\n\n"
-	"\t%d %s %s %s %s %s %s\n"
-	"\t|  |  |  |  |  |  |\n"
-	"\t|  |  |  |  |  |  +- \"connect\" or \"reuse\"\n"
-	"\t|  |  |  |  |  +---- Local port\n"
-	"\t|  |  |  |  +------- Local address\n"
-	"\t|  |  |  +---------- Remote port\n"
-	"\t|  |  +------------- Remote address\n"
-	"\t|  +---------------- Backend display name\n"
-	"\t+------------------- Connection file descriptor\n"
+	"\t%d %s %s %s %s %s %s [ %.6f %ld ]\n"
```

Comment:
I just realized we don't specify the precision in the manual, we simply go for `%f` (probably from a log consumer point of view).

Reply:
Yeah, good point. Did not realize we have not done this in the past.
I am leaning toward consistency, but having the precision allows you to select the correct data type to store this value.
Are we fine making such promises? or should I simply use `%f` here?

Reply:
We should put `%f` here to be consistent with the rest of the tags.